keras_core/ops/math.py

```diff
@@ -450,3 +509,193 @@ def fft2(a):
     if any_symbolic_tensors(a):
         return FFT2().symbolic_call(a)
     return backend.math.fft2(a)
+
+
+class RFFT(Operation):
+    def __init__(self, n=None):
+        super().__init__()
+        self.n = n
+
+    def compute_output_spec(self, x):
+        # We are calculating 1D RFFT. Hence, rank >= 1.
+        if len(x.shape) < 1:
+            raise ValueError(
+                f"Input should have rank >= 1. "
+                f"Received: input.shape = {x.shape}"
+            )
+
+        if self.n is not None:
+            new_last_dimension = self.n // 2 + 1
+        else:
+            new_last_dimension = x.shape[-1] // 2 + 1
+        new_shape = x.shape[:-1] + (new_last_dimension,)
+
+        return (
+            KerasTensor(shape=new_shape, dtype=x.dtype),
+            KerasTensor(shape=new_shape, dtype=x.dtype),
+        )
+
+    def call(self, x):
+        return backend.math.rfft(x, n=self.n)
+
+
+@keras_core_export("keras_core.ops.rfft")
+def rfft(x, n=None):
```

Comment:
Surely `n` should be something like `fft_length`? Do we have any equivalent argument anywhere else in the API today?

Reply:
Fixed.
I used `n` due to the naming convention from numpy but I think `fft_length` should be more clearer.